contents/blog/product-market-fit-game.md

```diff
@@ -0,0 +1,351 @@
+
+Start "closing the loop" with users. This is a simple process to improve your product:
+
+![product improvement loop](../images/blog/pmf-game-guide/product-loop.png)
+
+Closing the loop above is the way to cement your product-market fit because:
+
+1. **Your product will get better fast.** You should weight user feedback very heavily compared to your instincts around what to build. 
+
+2. **It will generate word of mouth growth.** The one thing you can compete on is speed – you can outperform any competitors by providing a ridiculously excellent experience to your early users, so they will tell their friends. 
+
+3. **Listening to users compounds.** Like a snowball rolling down a hill, the more you listen to users and act on their feedback, the more useful feedback they'll share with you. 
+
+#### Failure modes
+
+* **Users aren't activating.** This means although they _signed up_, but can't _configure_ / get started with your product, so they _never_ get any value from it. You have two options here:
+
+  1. Activate your users manually. The classic example is the founders of Stripe who [used to visit their first users in person](http://paulgraham.com/ds.html#Manual) to integrate their product for them. Do things that don't scale. The bonus of this approach is that you'll get deeper customer interaction, so you'll learn more and you'll get a better relationship which will help you get more feedback.
+  
+  2. Make your activation/onboarding simpler. Can you make it 10x simpler?
+
+* **You are getting conflicting feedback on what to build.** Go to Level 5, this will help clarify which users are your Ideal Customer Profile, and which aren't. Hopefully this exercise will help you pick a clearer direction.
+ 
+* **Your product isn't solving your users' problem**, hence there is no point coming back. Build a new product for the same problem, or fix your existing product. Restart level 4.
+
+* **Your product is painful to use.** Get feedback from users, look at session replays, or watch people trying to use it.
+
+* **Your users don't need to come back repeatedly.** Perhaps you've automatically solved the problem with no further user work. Proceed to Level 5 and validate this by seeing if they will pay for it. Don't force product engagement if it's irrelevant to solving the problem.
+
+* **Users use your product but don't encounter your problem often.** Proceed to Level 5. It may be hard to get significant revenue for an infrequent problem, or it may be that you don't experience much demand. If the number of users that experience this problem is huge even if the order value is tiny, that's ok. If it was extremely hard finding the first users _and_ it's hard to charge much for the product, this problem isn't a big enough one – go back to level 1.
```

Comment:
```suggestion
* **Users use your product but don't encounter your problem often.** Proceed to level 5. It may be hard to get significant revenue for an infrequent problem, or it may be that you don't experience much demand. If the number of users that experience this problem is huge even if the order value is tiny, that's ok. If it was extremely hard finding the first users _and_ it's hard to charge much for the product, this problem isn't a big enough one – go back to level 1.
```